Testing/dice_test.py

```diff
@@ -0,0 +1,12 @@
+import pytest
```

Comment:
Instead of a `dice_test` file, we might want to rename it to be `test_player` and then we can use this file to test the new player methods